editor/components/block-inspector/index.js

```diff
@@ -26,10 +25,7 @@ const BlockInspector = ( { selectedBlock, count } ) => {
 	}
 
 	return (
-		<div>
-			<Slot name="Inspector.Controls" />
-			<BlockInspectorAdvancedControls key={ `advanced-controls-${ selectedBlock.uid }` } />
-		</div>
+		<Slot name="Inspector.Controls" />
```

Comment:
It conflicts with the changes I propose with #3475, but we can resolve that later if necessary.